test/integration/validate-taskdefinition/badtaskdefinition.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: lifecycle.keptn.sh/v1alpha3
+kind: KeptnTaskDefinition
+metadata:
+  name: badtaskdefinition
+spec:
```

Comment:
please also add a comment about why this task definition will not be accepted by the validating webhook (i.e. function and container being present at the same time).
Also, I suggest to also add another "bad" taskDef where no container/function is present in the spec, to also cover that scenario

Reply:
Alright I will do that

Reply:
@bacherfl I have applied the suggestion.